### PR TITLE
New package: Jingque.Fanyiniao version 2.2.0

### PR DESCRIPTION
**Pull request by SpecterShell**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/160616)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Installation failed with exit code -1978335141
> 2024-06-29 00:51:16.708 [CLI ] Unable to locate nested installer at: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\Jingque.Fanyiniao.2.2.0\extracted\翻译鸟.exe

Automated error analysis suggests -1978335141 may mean Unable to locate nested installer. Does the `RelativeFilePath` have the previous version number in it?

(Automated response - build 886.)

**Comment by SpecterShell** [comment]
> Automatic Validation ended with:
> 
> > Installation failed with exit code -1978335141
> > 2024-06-29 00:51:16.708 [CLI ] Unable to locate nested installer at: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\Jingque.Fanyiniao.2.2.0\extracted\翻译鸟.exe
> 
> Automated error analysis suggests -1978335141 may mean Unable to locate nested installer. Does the `RelativeFilePath` have the previous version number in it?
> 
> (Automated response - build 886.)

The `RelativeFilePath` is correct. Is it an encoding issue that the filename in the zip file is not properly interpreted?

**Review by ItzLevvie — APPROVED** [review]

File: manifests/j/Jingque/Fanyiniao/2.2.0/Jingque.Fanyiniao.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+InstallerType: zip
+InstallerSwitches:
+  Upgrade: --updated
+Scope: user
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  NestedInstallerType: nullsoft # electron-builder
+  NestedInstallerFiles:
+  - RelativeFilePath: 翻译鸟.exe
```
Comment (stephengillie) [inline manifests/j/Jingque/Fanyiniao/2.2.0/Jingque.Fanyiniao.installer.yaml:15]:
```suggestion
  - RelativeFilePath: τ┐╗Φ»æΘ╕ƒ.exe
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @SpecterShell,

Manual Validation also ended with:
> 
> > Installation failed with exit code -1978335141
> > 2024-06-29 00:51:16.708 [CLI ] Unable to locate nested installer at: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\Jingque.Fanyiniao.2.2.0\extracted\翻译鸟.exe

The name appears to misencode into `τ┐╗Φ»æΘ╕ƒ`. 

Verify:
image

But when replacing the characters with their misencoding, the package manager is able to find the nested installer file. This isn't the best solution - how should this PR proceed?

**Comment by SpecterShell** [comment]
> Hi @SpecterShell,
> 
> Manual Validation also ended with:
> 
> > > Installation failed with exit code -1978335141
> > > 2024-06-29 00:51:16.708 [CLI ] Unable to locate nested installer at: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\Jingque.Fanyiniao.2.2.0\extracted\翻译鸟.exe
> 
> The name appears to misencode into `τ┐╗Φ»æΘ╕ƒ`.
> 
> Verify: image
> 
> But when replacing the characters with their misencoding, the package manager is able to find the nested installer file. This isn't the best solution - how should this PR proceed?

This might be a filename encoding issue of the zip archive. I would prefer to close the PR as it may result in errors on devices with different administrative locales.

**Pull request closed without merging**